### PR TITLE
Updates to internationalisation

### PR DESCRIPTION
**Pull request by rhodricusack**
Translations from Many Babies at Home project, with first (very nearly) complete translation, Japanese.

In this pull request:

1. Added to .dockerignore directory "data", where postgresql stores database
2. Added to INSTALLED_APPS in settings "django_countries", so that internationalisation is picked up (thanks to https://stackoverflow.com/a/25105221)
3. Added various missing translation tags

### CODE REVIEW

**Comment by becky-gilbert** [comment]
Thanks @rhodricusack and @shotsuji! 

@mekline I'm adding this to the shaping column on the project board.

**Review by okaycj — APPROVED** [review]
Wow!  LGTM.  Tests and linting pass.  Thank you for taking the time to put in these changes.

File: accounts/forms.py
Comment (mekline) [inline accounts/forms.py:344]: @okaycj Can you check version control on this vs. current production? Just confused why this is showing up as an addition in this PR

**Comment by mekline** [comment]
@rhodricusack @shotsuji This is so exciting to see! Once this PR is finished the results will appear at lookit-staging.mit.edu, and it would be ideal to get one of you/your team to review the results for accuracy before we deploy to production. Would that work for you?

**Comment by rhodricusack** [comment]
Yes, that would be great, thanks Melissa.


Rhodri Cusack
Director, Trinity College Institute of Neuroscience
The Thomas Mitchell Professor of Cognitive Neuroscience
School of Psychology
Trinity College Dublin
Dublin 2
Ireland

Learn about TCIN www.tcd.ie/neuroscience
Overview of my group: www.cusacklab.org
Infant fMRI: www.foundcog.org
Online tasks for infants: baby.cusacklab.org

________________________________

**Comment by rhodricusack** [comment]
Do I need to do anything further or shall I just wait for you to merge?

**Comment by becky-gilbert** [comment]
@rhodricusack sorry for the delay! No we're not waiting on anything from you. I will try to get to this today.

@okaycj I'd like to close the other PR (with linting changes) in favor of this one and then add the linting changes during/after merging.

**Comment by becky-gilbert** [comment]
@rhodricusack this is now live on the staging site! Please feel free to review it there and let us know if you run into any problems.

**Pull request merged**
